src/main/apiClient.ts

```diff
@@ -484,6 +485,16 @@ export default class ApiClient {
     })
   }
 
+  sendVerificationEmail(params: EmailVerificationParams): Promise<void> {
+    const { accessToken, ...data } = params
+    return this.http.post('/send-email-verification', { body: { ...data }, accessToken })
+  }
+
+  sendVerificationSms(params: { accessToken: string }): Promise<void> {
```

Comment:
You could create a `SmsVerificationParams` type (easier for documentation).

Reply:
Yes I thought about that, but for some reason decided not to. Will do.